lib/buffer.js

```diff
@@ -590,11 +589,8 @@ Buffer.prototype.toString = function(encoding, start, end) {
 
     if (end <= start)
```

Comment:
@benjamingr Not sure, what are you saying? :smile: I didn’t really touch this part of the logic ;)